CMakeLists.txt

```diff
@@ -429,90 +388,92 @@ if(MSVC)
     add_definitions(-D_CRT_NONSTDC_NO_DEPRECATE)
 endif()
 
-# Check whether compiler supports SSE2 instrinics
-if(WITH_NATIVE_INSTRUCTIONS)
-    set(CMAKE_REQUIRED_FLAGS "${NATIVEFLAG}")
-else()
-    set(CMAKE_REQUIRED_FLAGS "${SSE2FLAG}")
-endif()
-check_c_source_compile_or_run(
-    "#include <immintrin.h>
-    int main(void)
-    {
-        __m128i zero = _mm_setzero_si128();
-        (void)zero;
-        return 0;
-    }"
-    HAVE_SSE2_INTRIN
-)
-set(CMAKE_REQUIRED_FLAGS)
-
-# Check whether compiler supports SSE4 CRC inline asm
-if(WITH_NATIVE_INSTRUCTIONS)
-    set(CMAKE_REQUIRED_FLAGS "${NATIVEFLAG}")
-else()
-    set(CMAKE_REQUIRED_FLAGS "${SSE4FLAG}")
-endif()
-check_c_source_compile_or_run(
-    "int main(void)
-    {
-        unsigned val = 0, h = 0;
-    #if defined(_MSC_VER)
-        { __asm mov edx, h __asm mov eax, val __asm crc32 eax, edx __asm mov val, eax }
-    #else
-        __asm__ __volatile__ ( \"crc32 %1,%0\" : \"+r\" (h) : \"r\" (val) );
-    #endif
-        return (int) h;
-    }"
-    HAVE_SSE42CRC_INLINE_ASM
-)
-# Check whether compiler supports SSE4 CRC instrinics
-check_c_source_compile_or_run(
-    "#include <immintrin.h>
-    int main(void)
-    {
-        unsigned crc = 0;
-        char c = 'c';
-    #if defined(_MSC_VER)
-        crc = _mm_crc32_u32(crc, c);
-    #else
-        crc = __builtin_ia32_crc32qi(crc, c);
-    #endif
-        (void)crc;
-        return 0;
-    }"
-    HAVE_SSE42CRC_INTRIN
-)
-set(CMAKE_REQUIRED_FLAGS)
+if("${BASEARCH}" MATCHES "x86")
```

Comment:
Should be BASEARCH_X86_FOUND?

Reply:
mismerge, thanks

Reply:
This one still doesn't show yet.

Reply:
Doh, I tried adding it to the wrong commit, no wonder it was confused. It's in now.